File: hard/security/security.java
package ejercicios;

import java.util.Scanner;

/**
 *07/05/2021
 * @author Andre Peralta
 */
public class metodo {
    public static void main(String[] args) { 

        Scanner input = new Scanner(System.in);
        
        System.out.println("Floor : ");
        String floor = input.next();

        int money_pos = floor.indexOf("$");
	int thief_pos = floor.indexOf("T");

	if((money_pos - thief_pos) < 0){
		floor = floor.substring(money_pos,thief_pos+1); //[::-1]
		if(floor.indexOf('G') == -1){
                
                    System.out.println("ALARM");
                }
                else{
                    System.out.println("quiet");
                }

        }else{
		floor = floor.substring(thief_pos,money_pos+1);
		if(floor.indexOf('G') == -1){
                    
			System.out.println("ALARM");
                } 
                else{
			System.out.println("quiet");
                  }
        }
    }
}

File: hard/security/security.py
def alarm(floor):

	money_pos = floor.find('$')
	thief_pos = floor.find('T')

	if (money_pos - thief_pos) < 0:
		floor = floor[money_pos:thief_pos+1][::-1]
		if floor.find('G') == -1:
			print('ALARM')
		else:
			print('quiet')

	else:
		floor = floor[thief_pos:money_pos+1]
		if floor.find('G') == -1:
			print('ALARM')
		else:
			print('quiet')


floor = input()

alarm(floor)
